lib/pg_query/deparse.rb

```diff
@@ -22,7 +22,7 @@ def from(item)
 
     def deparse_item(item, context = nil) # rubocop:disable Metrics/CyclomaticComplexity
       return if item.nil?
-      return item if item.is_a?(Fixnum)
+      return item if item.is_a?(Integer)
```

Comment:
Do we need a version-check for the Ruby version here?

(this might not be tested by the test suite right now)